lang/php/projects.txt

```diff
@@ -0,0 +1,3 @@
+# Git URLs of publicly-accessible projects to be used for parsing stats,
+# one per line.
+#
```

Comment:
@mjambon could you polulate this? I tried the script but it has asking for a github API key.

Reply:
actually I just copy-pasted the one we had in semgrep/parsing-stats/php/

Reply:
yeah you have to generate an api key for yourself, preferably with just the read permission so you can copy-paste it in your shell without worrying.